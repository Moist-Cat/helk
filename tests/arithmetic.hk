print(2 + 3 * 4);
print((2 + 3) * 4);
print(10 / 2);
