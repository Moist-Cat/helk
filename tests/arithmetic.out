14.000000
20.000000
5.000000